Sources/SwiftProtobuf/MessageExtension.swift

```diff
@@ -19,22 +19,22 @@
 /// pieces.
 public protocol MessageExtensionBase {
     var fieldNumber: Int { get }
```

Comment:
I wasn't completely sure here, I'm guessing we want to expose the `fieldNumber` and `messageType`?